docs/user/filesystem.md

```diff
@@ -894,3 +894,126 @@ $
 
 While this mode is not expected to be used directly by users, it is useful for
 the unified namespace integration.
+
+## Interception Library `libpil4dfs`
+
+A new interception library called `libpil4dfs` is introduced. It is similar to libioil,
```

Comment:
please don't use new, introduced, etc. just speak about libpil4dfs. these are not release notes and should not talk about introducing something in a particular release as this lives with the source code.

Reply:
Revised. Thank you!